api/geth_backend.go

```diff
@@ -2079,8 +2085,8 @@ func (b *GethStatusBackend) startNode(config *params.NodeConfig) (err error) {
 		}
 	}()
 
-	b.log.Info("status-go version details", "version", params.Version, "commit", params.GitCommit)
-	b.log.Debug("starting node with config", "config", config)
+	b.logger.Info("status-go version details", zap.String("version", params.Version), zap.String("commit", params.GitCommit))
+	b.logger.Debug("starting node with config", zap.String("config", config.String()))
```

Comment:
zap.Stringer can be used to avoid having to call .String() when logging a value!

Reply:
Thank you, I forgot it exits, changed.